inappmessaging/src/test/java/com/rakuten/tech/mobile/inappmessaging/runtime/view/BaseViewSpec.kt

```diff
@@ -91,17 +90,49 @@ class BaseViewSpec : BaseTest() {
         `when`(mockBtn.buttonTextColor).thenReturn("test")
         `when`(mockBtn.buttonTextColor).thenReturn("#")
         `when`(mockBtn.buttonBackgroundColor).thenReturn("#")
-        view?.populateViewData(mockMessage, 1f)
+        view?.populateViewData(mockMessage)
         val button = view?.findViewById<MaterialButton>(R.id.message_single_button)
         button?.textColors shouldBeEqualTo ColorStateList.valueOf(Color.parseColor("#1D1D1D"))
         button?.backgroundTintList shouldBeEqualTo ColorStateList.valueOf(Color.WHITE)
     }
 
+    @Test
+    fun `should display image`() {
+        val imageUrl =
+            "https://iamprodjpefiles.blob.core.windows.net/campaign-images/e16c43ba-c4e2-491c-95d0-9444a6d63c91"
```

Comment:
Avoid using the actual URL of campaign images.

Reply:
Changed